pandaskill/app/player_team_page.py

```diff
@@ -0,0 +1,219 @@
+import datetime as dt
+import pandas as pd
+import altair as alt
+import numpy as np
+
+def display_player_team_page(data):
+    st.header("Player / Team Evolution")
+
+    data = data.reset_index()
+
+    data.rename(
+        columns={
+            "skill_rating_after_mu": "skill_rating_mu", 
+            "skill_rating_after_sigma": "skill_rating_sigma", 
+            "skill_rating_after": "skill_rating",
+            "performance_score": "pscore"
+        }, 
+        inplace=True
+    )
+
+    settings_columns = st.columns([1, 9])
+    with settings_columns[0]:
+        selection_type = st.selectbox("Select Type", ["Player", "Team"])
+
+    with settings_columns[1]:
+        if selection_type == "Player":
+            all_players = data[["player_id", "player_name"]].drop_duplicates()
+            player_name_to_id = all_players.set_index("player_name").to_dict()["player_id"]
+            player_names = list(player_name_to_id.keys())
+            selected_player_names = st.multiselect(
+                "Select up to two players to compare:",
+                player_names,
+                default=["Faker"] if "Faker" in player_names else [],
+                max_selections=5 
+            )
+            if len(selected_player_names) == 0:
+                st.warning("Please select at least one player.")
+                return
+            selected_player_ids = [player_name_to_id[name] for name in selected_player_names]
+            ratings = data.loc[data["player_id"].isin(selected_player_ids)].copy()
+            ratings['entity_name'] = ratings['player_name']
+        elif selection_type == "Team":
+            all_teams = data[["team_id", "team_name"]].drop_duplicates()
+            team_name_to_id = all_teams.set_index("team_name").to_dict()["team_id"]
+            team_names = list(team_name_to_id.keys())
+            selected_team_names = st.multiselect(
+                "Select up to two teams to compare:",
+                team_names,
+                default=["T1"] if "T1" in team_names else [],
+                max_selections=5
+            )
+            if len(selected_team_names) == 0:
+                st.warning("Please select at least one team.")
+                return
+            selected_team_ids = [team_name_to_id[name] for name in selected_team_names]
+            team_data = data.loc[data["team_id"].isin(selected_team_ids)].copy()
+            team_data["entity_name"] = team_data["team_name"]
+            ratings = team_data.groupby(["entity_name", "game_id", "date"]).agg(
+                serie_name=("serie_name", "first"),
+                tournament_name=("tournament_name", "first"),
+                pscore=("pscore", "mean"),
+                skill_rating_mu=("skill_rating_mu", "mean"),
+                skill_rating_sigma=("skill_rating_sigma", lambda x: np.sqrt(np.mean(np.square(x)))),
+            ).reset_index()
+            ratings["skill_rating"] = ratings["skill_rating_mu"] - 3 * ratings["skill_rating_sigma"]
```

Comment:
also I feel like it's missing line breaks in this function. For instance this line, computing actual skill rating from mu and sigma could be highlighted with a line break (in addition to a comment)